subprojects/docs/src/snippets/buildCache/cacheable-bundle-task/groovy/build.gradle

```diff
@@ -7,10 +7,15 @@ abstract class NpmTask extends DefaultTask {
 
     ListProperty<String> args = project.objects.listProperty(String)
 
+    @Internal
+    protected File bundleJSFile = project.file("$project.buildDir/bundle.js")
+    @Internal
+    protected File[] scriptsFiles = project.file("scripts").listFiles()
```

Comment:
🔴 This changes semantics as the `listFiles()` call would now happen at task instantiation time and not at task execution time.

Reply:
Sure, fixed I guess